nym-api/src/nym_nodes/handlers/unstable/mod.rs

```diff
@@ -85,6 +85,11 @@ struct NodesParamsWithRole {
     no_legacy: Option<bool>,
     page: Option<u32>,
     per_page: Option<u32>,
+
+    // Identifier for the current epoch of the topology state. When sent by a client we can check if
+    // the client already knows about the latest topology state, allowing a `no-updates` response
+    // instead of wasting bandwidth serving an unchanged topology.
+    epoch_uid: Option<u32>,
```

Comment:
just a nit: in all other places in our code base, we use `epoch_id` rather than `epoch_uid`